src/validation/ndrop/explmix.cpp

```diff
@@ -38,7 +38,7 @@ void explmix(Ensemble *ensemble) {
 
     for (int k = top_lev; k < pver; k++) {
       // add logic for km1 and kp1 from fortran
-      int kp1 = haero::min(k + 1, pver);
+      int kp1 = haero::min(k + 1, pver - 1);
```

Comment:
Is this what's done in the original Fortran (aside from the 0-based/1-based difference, that is)? This will prevent an out-of-bounds error, but it means that `k` and `kp1` are the same when `k == pver-1`.

Reply:
That is a good question - the initial snippet from Fortran looks like this:
```
    do kk=top_lev,pver
       kp1=min(kk+1,pver)
       km1=max(kk-1,top_lev)
```
so I think with Fortran vs C++ indexing the added -1 makes sense?